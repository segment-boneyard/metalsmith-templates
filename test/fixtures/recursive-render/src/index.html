---
template: secondary.html
title: title from src file
---

body
