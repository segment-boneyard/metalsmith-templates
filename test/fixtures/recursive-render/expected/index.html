<html>
<head>
  <title>title from src file</title>
</head>
<body>
<p>This is some content from the secondary template.</p>

body

<p>More content from the secondary template.</p>

</body>
</html>
